check.go

```diff
@@ -12,18 +14,20 @@ import (
 
 // Check details.
 type Check struct {
-	Name   string `json:"name,omitempty"`
-	Web    string `json:"web,omitempty"`
-	Shell  string `json:"shell,omitempty"`
-	Match  string `json:"-"`
-	Return int    `json:"-"`
-	Notify string `json:"-"`
-	Alert  string `json:"-"`
-	Tries  int    `json:"-"`
-	Repeat int    `json:"-"`
-	Sleep  int    `json:"-"`
-	Failed bool   `json:"failed" yaml:"-"`
-	Since  string `json:"since,omitempty" yaml:"-"`
+	Name    string            `json:"name,omitempty"`
+	Web     string            `json:"web,omitempty"`
+	Headers map[string]string `json:"headers,omitempty"`
+	Body    string            `json:"body,omitempty"`
```

Comment:
Ah, I was wondering why there were JSON tags on everything, and YAML tags on almost nothing.  I thought the yaml library might be doing something "clever."

You're right, there would be no purpose to the json tags.  

You can reject the entire pull request; if I get time to poke at it again I'll submit a new one.